bridge/github/config.go

```diff
@@ -205,13 +206,61 @@ func promptUsername() (string, error) {
 	}
 }
 
-func promptURL() (string, string, error) {
+func promptURL(repo repository.RepoCommon) (string, string, error) {
 	for {
-		fmt.Print("Github project URL: ")
 
-		line, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		fmt.Println("Detected Projects:")
+
+		repos, nil := repo.ListRemotes()
+		reposMap := make(map[int]string)
+		i := 1
+		for _, value := range repos {
+
+			if checkGithubURL(value) {
+				shortenedURL, err := shortenedURL(value)
+				if err != nil {
+					fmt.Println(err)
+					continue
+				}
+				fmt.Printf("[%d]: %s\n", i, shortenedURL)
+				reposMap[i] = value
+				i++
+			}
+		}
+
+		if i == 1 {
+			fmt.Printf("No github projects found\n")
+		}
+
+		fmt.Printf("\n[0]: Another project\n")
+		fmt.Println()
+		var line string
+		output, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		output = strings.TrimRight(output, "\n")
+
+		outputInteger, _ := strconv.Atoi(output)
+
 		if err != nil {
-			return "", "", err
+			fmt.Println(err)
+			return "", "", fmt.Errorf("Input error")
+		}
+
+		if outputInteger == 0 {
+			fmt.Println("Enter Project URL:")
+			line, err = bufio.NewReader(os.Stdin).ReadString('\n')
+			line = strings.TrimRight(line, "\n")
+
+			if !checkGithubURL(line) {
+				fmt.Printf("Please Enter valid github URL\n")
+				fmt.Println()
+				continue
+			}
+
+			if err != nil {
+				return "", "", fmt.Errorf("Input error")
```

Comment:
Go errors should not be capitalised: https://github.com/golang/go/wiki/CodeReviewComments#error-strings